sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1537,6 +1537,18 @@ object SQLConf {
       .booleanConf
       .createWithDefault(false)
 
+  val V2_BUCKETING_ALLOW_COMPATIBLE_TRANSFORMS =
+    buildConf("spark.sql.sources.v2.bucketing.allow.enabled")
```

Comment:
may be rename `spark.sql.sources.v2.bucketing.allow.enabled` to `spark.sql.sources.v2.bucketing.allowCompatibleTransform.enabled`

Reply:
Thanks, yea it is a copy and paste error.